tests/integration/targets/azure_rm_rediscache/tasks/main.yml

```diff
@@ -489,6 +529,22 @@
       azure_rm_rediscache:
         resource_group: "{{ resource_group }}"
         name: "{{ redis_name }}2"
+        sku:
+          name: premium
+          size: P1
         state: absent
       register: output
       tags: [long_run, never]
+
+- name: Destroy User Managed Identities
+  azure_rm_resource:
+    resource_group: "{{ resource_group }}"
+    provider: ManagedIdentity
+    resource_type: userAssignedIdentities
+    resource_name: "{{ item }}"
+    api_version: "2023-01-31"
+    state: absent
+  loop:
+    - "ansible-test-redis-identity"
+    - "ansible-test-redis-identity-2"
+
```

Comment:
Too many blank lines, Please delete the blank lines!
```suggestion
```

Reply:
Thanks, Done.